From: Greg Kroah-Hartman <gregkh@suse.de>
Date: Thu, 25 Aug 2011 10:25:37 -0700
Patch-mainline: v3.2-rc1
Subject: [PATCH] Staging: hv: add driver_data to hv_vmbus_device_id
Git-commit: a91befc116f424883602fb7b57b63b5fcfe18719

This is going to be needed by some drivers that handle more than one
device, like all other bus types do, so prepare for that in advance
before the user/kernel api is used.

Cc:  K. Y. Srinivasan <kys@microsoft.com>
Cc: Haiyang Zhang <haiyangz@microsoft.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>
---
 include/linux/mod_devicetable.h |    2 ++
 1 files changed, 2 insertions(+), 0 deletions(-)

diff --git a/include/linux/mod_devicetable.h b/include/linux/mod_devicetable.h
index 5a12377..468819c 100644
--- a/include/linux/mod_devicetable.h
+++ b/include/linux/mod_devicetable.h
@@ -410,6 +410,8 @@ struct virtio_device_id {
  */
 struct hv_vmbus_device_id {
 	__u8 guid[16];
+	kernel_ulong_t driver_data	/* Data private to the driver */
+			__attribute__((aligned(sizeof(kernel_ulong_t))));
 };
 
 /* i2c */
-- 
1.6.0.2


